scripts/hooks/hooks.yml

```diff
@@ -0,0 +1,19 @@
+- id: deploy
+  execute-command: ./deploy.sh
+  include-command-output-in-response: true
+  pass-environment-to-command:
+      - source: url
```

Comment:
I think we should move all this to the body of the request, so it isn't available to read on the URL in transit.  Best to hide as much as we can via SSL.

Reply:
Agreed, having secrets in readable URLs irks me haha

Reply:
[Resolved.](https://github.com/Seneca-CDOT/starchart/pull/57/files#diff-609a3b67b49141c106c5e8c1326bdbca40532edf252cfc87fb7da5cde58af70bR6) All the params are now sourced from payload, instead of the url.